pkg/control/pod_control_test.go

```diff
@@ -33,7 +33,7 @@ import (
 	"k8s.io/kubernetes/pkg/api/legacyscheme"
 	"k8s.io/kubernetes/pkg/api/testapi"
 
-	"github.com/kubeflow/tf-operator/pkg/common/util/testutil"
+	"github.com/kubeflow/tf-operator/pkg/common/util/v1beta2/testutil"
```

Comment:
in the future, we have to restructure these tests. Common control package is currently using version specific testutil.

Reply:
Any way we can do this without duplicating common packages?

Reply:
Control packages are generic. It is not tied to TFJob. However, we added TF specific tests in control test packages. 
Also, test utils package can be restructured to avoid duplication of entire code. We can take this  up later.

Reply:
Opened https://github.com/kubeflow/tf-operator/issues/938 to track this.